core/assets/bundles/bundle_pt_BR.properties

```
@@ -2089,41 +2089,41 @@ unit.evoke.description = Constrói estruturas para defender o Bastião do Núcle
-lst.radar = Locate units around a building with range.
-lst.sensor = Get data from a building or unit.
-lst.set = Set a variable.
-lst.operation = Perform an operation on 1-2 variables.
-lst.end = Jump to the top of the instruction stack.
-lst.wait = Wait a certain number of seconds.
-lst.stop = Halt execution of this processor.
-lst.lookup = Look up an item/liquid/unit/block type by ID.\nTotal counts of each type can be accessed with:\n[accent]@unitCount[] / [accent]@itemCount[] / [accent]@liquidCount[] / [accent]@blockCount[]
-lst.jump = Conditionally jump to another statement.
-lst.unitbind = Bind to the next unit of a type, and store it in [accent]@unit[].
-lst.unitcontrol = Control the currently bound unit.
-lst.unitradar = Locate units around the currently bound unit.
-lst.unitlocate = Locate a specific type of position/building anywhere on the map.\nRequires a bound unit.
-lst.getblock = Get tile data at any location.
-lst.setblock = Set tile data at any location.
-lst.spawnunit = Spawn unit at a location.
-lst.applystatus = Apply or clear a status effect from a unit.
-lst.spawnwave = Spawn a wave.
-lst.explosion = Create an explosion at a location.
-lst.setrate = Set processor execution speed in instructions/tick.
-lst.fetch = Lookup units, cores, players or buildings by index.\nIndices start at 0 and end at their returned count.
-lst.packcolor = Pack [0, 1] RGBA components into a single number for drawing or rule-setting.
-lst.setrule = Set a game rule.
-lst.flushmessage = Display a message on the screen from the text buffer.\nWill wait until the previous message finishes.
-lst.cutscene = Manipulate the player camera.
-lst.setflag = Set a global flag that can be read by all processors.
-lst.getflag = Check if a global flag is set.
+lst.read = Lê um numero de uma célula de memoria conectada.
+lst.write = Escreve um numero em uma célula de memória conectada.
+lst.print = Adiciona texto ao buffer de impressão.\nNão vai exibir nada até o [accent]Print Flush[] ser usado.
+lst.draw = Adiciona uma operação ao buffer de desenho.\nNão vai exibir nada até o [accent]Draw Flush[] ser usado.
+lst.drawflush = Libera operações de [accent]Draw[] em fila para ser exibida.
+lst.printflush = Libera operações de [accent]Print[] em fila para um bloco de mensagem.
+lst.getlink = Pega uma conexão do processador pelo index. Começa no 0.
+lst.control = Controla uma construção.
+lst.radar = Localiza unidades ao redor de uma construção com alcance.
+lst.sensor = Pega dados de uma construção ou unidade.
+lst.set = Define uma variável.
+lst.operation = Executa uma operação em 1-2 variáveis.
+lst.end = Pule para o topo da lista de instruções.
+lst.wait = Espere um certo número de segundos.
+lst.stop = Interrompa a execução deste processador.
+lst.lookup = Procura um item/líquidp/unidade/bloco pelo ID.\nContagem total de cada tipo pode ser acessada com:\n[accent]@unitCount[] / [accent]@itemCount[] / [accent]@liquidCount[] / [accent]@blockCount[]
```

- "líquidp" should be "líquido"
- "bloco pelo ID" as "tipo de bloco por ID"
```suggestion
lst.lookup = Procura um item/líquido/unidade/tipo de bloco por ID.\nContagem total de cada tipo pode ser acessada com:\n[accent]@unitCount[] / [accent]@itemCount[] / [accent]@liquidCount[] / [accent]@blockCount[]
```